test/Xunit.StaFact.Tests.Mac/Xunit.StaFact.Tests.Mac.csproj

```diff
@@ -20,7 +20,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="xunit.v3" />
```

Comment:
ℹ️ `xunit.v3.core` v2 includes a check that the project is an executable and includes the .NET app host:
https://github.com/xunit/xunit/blob/2b9c9e2005ebed10c692e03de0c85fe3857b1180/src/xunit.v3.core/Package/buildTransitive/xunit.v3.core.targets#L13-L17

I have a feeling that [`UseAppHost`](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#useapphost) needs to be false for this project since it has its own entry point. Rather than referencing `xunit.v3` which references `xunit.v3.core`, I've changed the references to include the specific packages that are needed and to not reference `xunit.v3` or `xunit.v3.core` so that the `.targets` file from `xunit.v3.core` that performs that check is not included.